docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java

```diff
@@ -175,6 +175,11 @@ protected boolean isCcrTest() {
         return testName != null && testName.contains("/ccr/");
     }
 
+    protected boolean isGetLicenseTest() {
+        String testName = getTestName();
+        return testName != null && (testName.contains("/get-license/") || testName.contains("\\get-license\\"));
```

Comment:
Why does this need to support `\\` but ccr doesn't?

Reply:
My guess is that ccr tests do not run on Windows? There are other similar methods in this file, e.g. `isMachineLearningTest`, `isTransformTest` and they all check for the backslash'd variant. The CI does have machine learning specific jobs so I think they are justified. 

For license check, I decided to add it because we do have multijob-windows-compatibility jobs. I am not completely sure that these jobs run the license check. But it's easy to add so I just did it.